stix_shifter_modules/qradar/stix_translation/json/to_stix_map.json

```diff
@@ -459,6 +459,21 @@
       "object": "x-qradar"
     }
   ],
+  "dnsdomainname": [
+    {
+      "key": "url.value",
+      "object": "url"
+    },
+    {
+      "key": "domain-name.value",
+      "object": "domain1",
+      "transformer": "ToDomainName"
+    },
+    {
+      "key": "x-qradar.domain_name",
+      "object": "x-qradar"
```

Comment:
you don't need `x-qradar.domain_name` as you already mapped to standard domain name object